### PR TITLE
build: Generate index.html for publication, package modules

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

**Comment by thewtex** [comment]
We need to generate an index.html for #6 . This produces dist/index.html with the updated script imports.

Also some configuration towards #11 . I am not sure how we want to set up package `exports`. Perhaps we need another package for the custom element or an index module that exports both.

File: vite.config.ts [hunk 1]
```diff
@@ -1,14 +1,40 @@
...
+      name: 'ItkViewer',
+      fileName: (format, entryName) => {
+        const extension = format === 'es' ? '.js' : '.umd.js'
+        switch(entryName) {
+        case 'lib':
+          return `itk-viewer${extension}`
+        case 'app':
+          return `itk-viewer-app${extension}`
+        case 'element':
+          return `itk-viewer-element${extension}`
```
Comment (PaulHax) [inline vite.config.ts:22]: Do we need to prefix?

Comment (thewtex) [reply vite.config.ts:22]: Yeah, I think we should just put this in another package.

Comment (PaulHax) [reply vite.config.ts]: Just curious: why do we put itk-viewer in front?

**Review by PaulHax — APPROVED** [review]

File: vite.config.ts (hunk 1 above)
Comment (thewtex) [reply vite.config.ts]: Module name corresponds to the name of the package

**Pull request merged**